frontend/src/components/elements/Text/Text.scss

```diff
@@ -89,5 +89,12 @@
       color: inherit;
       text-decoration: underline;
     }
+
+    div.code-block {
+      button.overlayBtn {
+        top: 0;
+        right: 0;
+      }
+    }
```

Comment:
It would be nice to find a solution that doesn't special-case `overlayBtn` inside `code-block`. Is it possible to make `overlayBtn` "just work" everywhere, so long as the container has `position` set to something (e.g. `relative`)?

Reply:
This is needed when we use a codeBlock inside an alert, if we use the same style the overlayBtn will be positioned wrong. We are using relative but the alert has some paddings that breaks that.